cli/pcluster/pcluster.py

```diff
@@ -65,7 +66,7 @@ def create_bucket_with_batch_resources(stack_name, aws_client_config, resources_
 
 def version(args):
```

Comment:
We have to pass it to the `version()` function in `cli.py` because argparser needs it to accept args. From there it's being passed through just to maintain consistency. 

I can remove `args` from `version()` in `pcluster.py`, then remove it from the function call [#L59](https://github.com/aws/aws-parallelcluster/blob/develop/cli/pcluster/cli.py#L59)

Reply:
done